### PR TITLE
feat(WebexRemoteMedia): crop media inside the meeting

### PR DESCRIPTION
**Pull request by CernaianuMihai**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-260359

### CODE REVIEW

File: src/styles/_mixins.scss [hunk 1]
```diff
@@ -70,3 +78,15 @@ $shadow-color: rgba(0, 0, 0, 0.2);
 
   z-index: 1;
 }
+
+@mixin tablet {
+  @media (min-width: #{$tablet-width}) {
```
Comment (cipak) [inline src/styles/_mixins.scss:83]: I don't think we can use media queries, our components can be much smaller than the screen. We need to use useElementDimensions to take the width and add classes conditionally.

Comment (CernaianuMihai) [reply src/styles/_mixins.scss]: I reversed these changes.

File: src/components/WebexInMeeting/WebexInMeeting.scss [hunk 2]
```diff
@@ -6,13 +6,19 @@ $shadow-color: rgba(0, 0, 0, 0.2);
...
 
   &.remote-sharing {
     .local-media-in-meeting {
       display: none;
     }
   }
 
+  .media-container {
+    position: relative;
+    flex-direction: column;
```
Comment (cipak) [inline src/components/WebexInMeeting/WebexInMeeting.scss:19]: add `display: flex` before `flex-direction`

File: src/components/WebexInMeeting/WebexInMeeting.scss [hunk 3]
```diff
@@ -21,6 +27,7 @@ $shadow-color: rgba(0, 0, 0, 0.2);
   }
 
   .remote-media-in-meeting {
+    height: 100%;
```
Comment (cipak) [inline src/components/WebexInMeeting/WebexInMeeting.scss:30]: just flex: 1 is enough if you add display: flex on parent

File: src/components/WebexInMeeting/WebexInMeeting.scss (hunk 2 above)
Comment (CernaianuMihai) [reply src/components/WebexInMeeting/WebexInMeeting.scss:20]: done

File: src/components/WebexInMeeting/WebexInMeeting.scss (hunk 3 above)
Comment (CernaianuMihai) [reply src/components/WebexInMeeting/WebexInMeeting.scss]: done

**Review by cipak — APPROVED** [review]

**Pull request merged**